code/modules/wiremod/components/math/trig.dm

```diff
@@ -0,0 +1,125 @@
+/**
+ * # Trig Component
+ *
+ * Math trig, not a trigger
+ * Does trig stuff
+ */
+
+ //Generic class for other trig components
+/obj/item/circuit_component/trig
```

Comment:
You should increase the power for these.

Reply:
I don't see any other component using the power_usage_per_input var, which would be the appropriate way to increase power usage for this component, so I don't really have a frame of reference in what I should increase it to. What value do you recommend?